integration_tests/models/unit_tests/test_path_transformations_actual.sql

```diff
@@ -0,0 +1,55 @@
+{% if target.type == 'bigquery' -%}
+
+  with raw_data as (
+
+      select * from {{ ref('test_path_transformations_data') }}
+
+    )
+
+    , arrays as (
+      select
+        transformed_path as raw_array,
+        {{ snowplow_fractribution.trim_long_path('transformed_path', 1) }} as trim_long_path,
+        {{ snowplow_fractribution.path_transformation('unique_path') }} as unique_path,
```

Comment:
@zloff I was meant to ask this already but forgot, is there any reason behind applying the trim_long_path() on the path only but not on the transformed_path? See [original code](https://github.com/snowplow/dbt-snowplow-fractribution/blob/96acd8a1124266ee3ad73169d08dd6a46fd668ac/models/snowflake/snowplow_fractribution_paths_to_conversion.sql#L11). Because of this I treat them separately in the model. Thanks.

Reply:
From what I understand, Mike stuck to the logic of the original sql when possible, looking at the code from Google, it seems that they did it this way: https://github.com/google/fractribution/blob/72ad8a4a851c75152c700283fbc4f58450b083af/py/templates/paths_to_conversion.sql#L25 
I couldn't say what the reasoning is though